Add doodles: Starwars day, TalkLikeAPirate day, Guy Fawkes day

salazarp opened the pull request:
A few harvey-doodles for the website.

elbing commented:
LGTM

sevki commented:
@salazarp can you change the file names to `mm/dd.ext` format? or the naming convention and the other files to be consistent

salazarp commented:
Yes, sorry about that. 
I have appended filenames with keywords to describe the resources better to the users.
The numbering scheme is handy for scripting and having a tidy dir though, good idea.
I have updated the README with the convention and renamed files accordingly.

sevki commented:
thanks @salazarp.
LGTM
 I'll merge this in if you'd like

salazarp commented:
Yes please, that'd be nice. Cheers.
